public/assets/scripts/Class/Card.js

```diff
@@ -0,0 +1,65 @@
+"use strict";
+import FetchManager from "../Service/FetchManager.js";
+
+class Card {
+  id = null;
+  potatoe = "";
+  gloves = false;
+  underwear = false;
+  crown = false;
+  imageUrl = "";
+  identifier = "";
+
+  constructor(identifier = 0) {
+    this.identifier = identifier;
+    this.mountCard();
+  }
+
+  async addToStack(stackId = 0) {
+    const data = {
+      stackId: stackId,
+      id: this.id,
+    };
+
+    const response = await FetchManager.post(
+      "/add-card-to-stack/" + stackId,
```

Comment:
```suggestion
      "/add-card-to-stack/",
```
C'est une requete POST, pas d'argument dans la route.